### PR TITLE
issue-5: Capture uncaught promises errors in PageManager dependencies load and controller wrapping

### PR DESCRIPTION
**Pull request by klam**

### CODE REVIEW

File: page-manager.js [hunk 1]
```diff
@@ -672,11 +677,11 @@ define([
...
                 var wrapperOptions = ('string' === typeof controllerInfo.controllerWrapper) ?
-                                     {} : controllerInfo.controllerWrapper;
-
-                return new Promise(function(resolve) {
-                    controller.__wrapper.wrap(resolve, wrapperOptions, alreadyInStack);
-                });
+                                     {} : controllerInfo.controllerWrapper,
+                    wrapperPromise = new Promise(function(resolve) {
+                                        controller.__wrapper.wrap(resolve, wrapperOptions, alreadyInStack);
+                                    });
```
Comment (andreybs11) [inline page-manager.js:683]: Missing catch > Logger.error

**Comment by andreybs11** [comment]
This pull was made against master. 
@jpgarbanzo : Shouldn't we apply the same versioning system as other ju-repos? Branch+Tag+Milestone

**Comment by klam** [comment]
Yes, we have to create a 1.0.0 release and then a working branch for ongoing pull requests. 
Will wait for @jpgarbanzo for green signal to merge to master in the meanwhile.

File: page-manager.js (hunk 1 above)
Comment (jpgarbanzo) [inline page-manager.js:683]: I could add it later, as I might need to make a little change to controller wrapper flow

**Pull request merged**